AgrI-Mitra/bff
# Encryption Decryption API Update

## Description

Updated the code.
Updated the Encryption and Decryption logic
Updated code with unique key function
Deprecated the PM KISAN ENC DEC API



### Changes


Uniquekey generation
Encryption Decryption within the code

## How to test
Valid aadhaar number.
Query to trigger the flow: "When will I get my installment?"
and then enter the Aadhaar number - otp and see the details.
